web/beacon-app/src/components/Error/ErrorPage/ErrorPage.tsx

```diff
@@ -1,16 +1,65 @@
+import { t } from '@lingui/macro';
 import { Button } from '@rotational/beacon-core';
-import { useRouteError } from 'react-router-dom';
-// error page with reload button
+import { isRouteErrorResponse, useRouteError } from 'react-router-dom';
 
-export default function ErrorPage() {
+import NotFoundOutters from '@/assets/images/not-found-outters.svg';
+import { Link } from '@/components/ui/Link';
+
+type ErrorPageProps = {
+  errorMessage?: string;
+  errorCause?: string;
+  errorTitle?: string;
+};
+
+export const render404 = () => {
+  return (
+    <section className="mx-auto my-20  flex max-w-4xl place-items-center items-center justify-center rounded-lg border border-solid border-primary-800 text-2xl">
+      <div className="my-10 mx-auto max-w-xl">
+        <h1 className="mt-4 text-2xl font-bold text-gray-800">
+          Sorry, we can’t find that page. (404)
+        </h1>
+        <p className="mt-4">
+          Return to
+          <Link href="/" className="text-primary-800">
```

Comment:
Should `text-primary` be used as the color here to be more consistent with how links appear on the site.